### PR TITLE
some typo errors were corrected. Kindly review

### PR DESCRIPTION
**Pull request by Esyteck**
I corrected typo errors in some of the markdown files. E.g pparticipants was corrected to participants

### CODE REVIEW

File: ols-2.md [hunk 1]
```diff
@@ -32,25 +32,26 @@ photos:
...
+{% capture all-hosts %}{{ all-hosts}}, {{ h }}{% endcapture %}
+{% endfor %}
+{% endif %}
+{% endfor %}
 {% endfor %}
 {% assign p-speakers = all-speakers | remove_first: ', ' | split: ", " | uniq | sort %}
 {% assign p-hosts = all-hosts | remove_first: ', ' | split: ", " | uniq | sort %}
 
 # The OLS-2 program
+
```
Comment (bebatut) [inline ols-2.md:54]: This empty line should stay so the `{:.no_toc}` is applied to it

File: ols-2.md [hunk 2]
```diff
@@ -72,9 +73,10 @@ and international bioinformatics communities.
 The vision of Open Life Science program is to strengthen Open Science skills for early stage researchers and young leaders in life science.
 
 At the end of the program, our participants will be able to:
-- Describe and define the terms *openness*, *open science*, *open leadership*, *community interactions*, *value exchanges*, *inclusivity*, *accessibility*, *open Science practices in developing resources and training*
+
+- Describe and define the terms _openness_, _open science_, _open leadership_, _community interactions_, _value exchanges_, _inclusivity_, _accessibility_, _open Science practices in developing resources and training_
 - Learn how to apply those principles to open leadership and working open in their projects and communities
-. Learn how to collect, invite, and tell stories that demonstrate how and why openness benefits the communities they serve
+  . Learn how to collect, invite, and tell stories that demonstrate how and why openness benefits the communities they serve
```
Comment (bebatut) [inline ols-2.md:79]: Why adding a space here?

File: _ols-1/projects-participants.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ photos:
 
 {% assign all-participants = '' %}
 {% for project in projects %}
-    {% assign p-pparticipants = '' %}
```
Comment (bebatut) [inline _ols-1/projects-participants.md:16]: It was not really a spelling mistake, mostly variable names

File: ols-2.md [hunk 4]
```diff
@@ -17,13 +17,13 @@ photos:
...
-    {% for m in project.mentors %}
-        {% capture all-mentors %}{{ all-mentors }}, {{ m }}{% endcapture %}
-    {% endfor %}
+{% assign p-participants = '' %}
+{% for p in project.participants %}
+{% capture all-participants %}{{ all-participants}}, {{ p }}{% endcapture %}
+{% endfor %}
+{% for m in project.mentors %}
+{% capture all-mentors %}{{ all-mentors }}, {{ m }}{% endcapture %}
+{% endfor %}
```
Comment (bebatut) [inline ols-2.md:26]: I would be nice to keep the indentation. It helps reading and understanding the large bunch of Jekyll nested code

**Review by bebatut — COMMENTED** [review]
Thanks @Esyteck for this 1st contribution
I put some comments in the code changes

File: ols-2.md (hunk 1 above)
Comment (Esyteck) [reply ols-2.md:54]: okay

File: ols-2.md (hunk 2 above)
Comment (Esyteck) [reply ols-2.md:79]: Not intentional. The editor on my pc did that. I'm sorry

File: _ols-1/projects-participants.md (hunk 3 above)
Comment (Esyteck) [reply _ols-1/projects-participants.md:16]: Okay. Thanks

File: ols-2.md (hunk 4 above)
Comment (Esyteck) [reply ols-2.md:26]: Not intentional. The editor on my pc did that. I'm sorry

**Comment by Esyteck** [comment]
Thanks a lot.

**Pull request merged**